### ISSUE TITLE
Add mp3 embedded album art option

### ISSUE THREAD

**Issue by cfredericksen** [dkanada/gelli#107]
Instead of showing album art stored in the folder add a setting to show the album art thats embedded in the mp3 file. Jellyfin native android app shows the art in the mp3 file.

**Comment by IceWreck** [comment]
Yeah Jellyfin has a strict folder structure for you music and if you dont follow it, then your library gets messed up. So mine is messed up and I get weird mixed album art, I'd appreciate this feature as well

**Comment by dkanada** [comment]
I'll need more information on this issue, I have absolutely no idea what you're referencing at the moment. Screenshots would be extremely helpful.

**Comment by cfredericksen** [comment]
The album art is shown from the folder.jpg in each album folder. MP3 files can also have album art embedded in the file itself as metadata.

**Comment by dkanada** [comment]
If this feature exists on any other clients, could I have a photo of an example or steps to reproduce the behavior? If this exists I'm not sure what exact situation is required to make it happen yet.

**Comment by cfredericksen** [comment]
I cant do a screenshot of the Jellyfin Android app. But attached is a screenshot of the embedded art thats in this mp3 file from the Jellyfin web GUI. I selected a song then 'Edit Images'. Does this help?  

![Screen Shot 2021-07-16 at 3 33 51 PM](https://user-images.githubusercontent.com/45824775/126005641-0ad69d6b-b99d-4f63-b001-9298ddadd5c0.png)
